### ISSUE TITLE
OCaml on AWS Lambda

### ISSUE THREAD

**Issue by Leonidas-from-XIV** [mads-hartmann/polyglots.dk#19]
AWS Lambda allows you to run Go and Java and Python and Node.js and apparently you can trick it to run Haskell.

Since I have a personal interest I'd be interested in trying to run OCaml code there, either via compiling to JS or native code. Pretty sure I could make a talk about my experiences with it.

**Comment by mads-hartmann** [comment]
@Leonidas-from-XIV When I was researching this it seemed people were using a tiny node shim to launch your binary. At least that seems to be how the Go people were doing it before it had native Go support :) Means you have to launch node yo launch your application, but at least it will run ;) I'd be happy to hear about your experience

**Comment by Leonidas-from-XIV** [comment]
Yes, that's also how the Haskell people used it, e.g. with http://apex.run/

But maybe the Go support can be piggybacked on. Or using `js_of_ocaml` though I would like to avoid that if possible.
